index.js

```diff
@@ -85,6 +97,59 @@ async function main (currentVersion) {
   }
 }
 
+/**
+ * get the currently supported Node versions
+ * @returns Promise<VersionInfo[]>
+ */
+function supportedVersions () {
+  return nv(['supported'])
+}
+
+/**
+ * @param {string} version
+ * @returns {Promise<number[]>} true if this version is a supported major version
+ */
+async function supportedMajorVersions () {
+  const versions = await supportedVersions()
+  const majors = versions.map(v => v.major)
+  return majors
+}
+
+/**
+ * @param {string} version
+ * @returns {Promise<boolean>} true if this version is a supported major version
+ */
+async function isNodeSupportedMajor (version) {
```

Comment:
We don't need this function anymore. `isNodeEOL` does all the trick. It covers:

- end-of-life versions
- invalid versions